build_push/action.yaml

```diff
@@ -0,0 +1,25 @@
+# This action expects a Docker username and password be stored as environment variables by the calling workflow
+
+name: phil_build_push
+inputs: 
+  name:
+    description: 'Name of project'
+    required: true
+  tag:
+    description: 'Tag to associate with built image'
+    required: true
+runs: 
+  using: "composite"
+  steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to DockerHub
+      uses: docker/login-action@v2
+      with:
+        username: $DOCKER_USERNAME
```

Comment:
```suggestion
        username: ${{ secrets.DOCKER_USERNAME }}
```

Reply:
This doesn't appear to work in reusable actions. The `secrets` object does not exist.

Reply:
Can we try this? https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/